### ISSUE TITLE
easyeffects support?

### ISSUE THREAD

**Issue by hitanol** [siddhpant/broo#6]
idk, im using broo a lot but i cant make it plug and play with easyeffects source :(
pls if you can make a workaround i'll be very grateful with you and the gods of wireless audio
(there was a previous issue buut it was more focused in obs, in this case the priority is linking broo to easyeffects mic source)
![imagen](https://user-images.githubusercontent.com/99027279/170840952-b88741f1-bdaf-40cc-8235-8d3b1642a0c6.png)

**Comment by siddhpant** [comment]
I'm busy for some weeks so will see later.

But as a workaround, maybe try applying effects on the Mumble output source itself.

**Comment by hitanol** [comment]
> Did you try it?

yes, since "Playback" and "loopback-####" output is connecting to EF Sink, i disconnect it and connect just Playback sink to Broo input, then connect Broo output to the first mic effect

![imagen](https://user-images.githubusercontent.com/99027279/179384256-153957ca-2bb2-4d1f-9a58-06a22fc047d9.png)

**Comment by siddhpant** [comment]
What I meant was: In EasyEffects, I can see an entry for loopback in the `Output` tab. Can you try that out? You won't need to do custom remapping that way, if it works.

![image](https://user-images.githubusercontent.com/25429745/179461836-a8d03cb2-a287-46d6-91f9-b44fb726c398.png)

**Comment by siddhpant** [comment]
TLDR: The problem maybe potentially happening because output of Mumble was moved due to EasyEffects. Bring it back to Broo using pavucontrol, and enjoy :)

**Comment by siddhpant** [comment]
> I don't see what problem you have, except possibly 

Bad sentence framing, that seems to be definitely the problem based on your earlier reply.

Sorry for not trying to use EasyEffects earlier.
